src/utilities/sequence-editor/sequence-completion.ts

```diff
@@ -151,10 +151,10 @@ export function sequenceCompletion(
               type: 'keyword',
             },
             {
-              apply: `${cursor.isAtSymbolBefore ? '' : '@'}GROUND_EPOCH("Name","+0.00") `,
+              apply: `G1 "epoch.name" `,
```

Comment:
are the "``" needed if there are no more dynamic strings here?

Reply:
They are not, would you like me to change it? For me it is just another way to represent a string.

Reply:
Eh. Yeah. It's probably not worth changing it. I just do it to be clearer about the intent of the string, but the string is so short here that it's pretty easy to immediately grok that it's constant.

Reply:
I snuck in the change now it is `'G1 "epoch.name" '`